openreview/invitations/content.py

```diff
@@ -24,8 +24,9 @@
     'comment': {
         'order': 1,
         'value-regex': '[\\S\\s]{1,5000}',
-        'description': 'Your comment or reply (max 5000 characters). Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$',
-        'required': True
+        'description': 'Your comment or reply (max 5000 characters). Write comment using Markdown and Latex formulas, more information: https://openreview.net/faq',
```

Comment:
@melisabok I am not sure if we support LaTeX or only plain TeX. I ask mainly because this line mentioned `TeX` earlier.

Reply:
@zbialecki @xkopenreview ?

Reply:
I would say, "Your comment or reply (max 5000 characters). Add formatting using Markdown and formulas using LaTeX. For more information see https://openreview.net/faq"

Reply:
Thanks, I made the changes.